DEVELOP.rst

```diff
@@ -147,9 +153,26 @@ To see stdout/stderr, use the following command:
 
     $ behave --no-capture
 
+Troubleshooting the integration tests
+-------------------------------------
+
```

Comment:
I think this paragraph belongs before the part about running behave tests. We need to make it clear tha it's a prerequisite to running tests. Something like this:

First, install the requirements... etc.
Then, make sure you have default database user `postgres` configured... etc.
Then:
`cd tests`
`behave`

Reply:
Would a pointer to the troubleshooting section do the work? Maybe like:

    First, install the requirements... etc.

    Make sure you have default database user postgres configured.
    **See the troubleshooting section for more information**

    Then:
        cd tests
        behave

Reply:
Troubleshooting has its place, perhaps, but the normal workflow should come first and take precedence. Yes, our tests are broken in a few places right now, but I don't want the documentation to "normalize" it. Tests being broken is not normal. You should not have trouble running them. I would describe how postgres user should be configured in "normal workflow" section. Then in "troubleshooting" section, you can add a few more pointers, like:
* make sure postgres instance on localhost is running
* check pg_hba config to verify local connections are enabled
* contact us on gitter or file an issue.